recipes/opuslib/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "opuslib" %}
+{% set version = "3.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2cb045e5b03e7fc50dfefe431e3404dddddbd8f5961c10c51e32dfb69a044c97
+
+build:
+  number: 0
```

Since this is pure python, this can be noarch python:
```suggestion
  number: 0
  noarch: python
```